src/rpc/util.cpp

```diff
@@ -66,3 +66,55 @@ CScript CreateMultisigRedeemscript(const int required, const std::vector<CPubKey
 
     return result;
 }
+
+UniValue DescribeAddressVisitor::operator()(const CNoDestination &dest) const {
+    return UniValue(UniValue::VOBJ);
+}
+
+UniValue DescribeAddressVisitor::operator()(const CKeyID &keyID) const {
+    UniValue obj(UniValue::VOBJ);
+    CPubKey vchPubKey;
+    obj.push_back(Pair("isscript", false));
+    obj.push_back(Pair("iswitness", false));
+    return obj;
+}
+
+UniValue DescribeAddressVisitor::operator()(const CScriptID &scriptID) const {
+    UniValue obj(UniValue::VOBJ);
+    CScript subscript;
+    obj.push_back(Pair("isscript", true));
+    obj.push_back(Pair("iswitness", false));
+    return obj;
+}
+
+UniValue DescribeAddressVisitor::operator()(const WitnessV0KeyHash& id) const
+{
+    UniValue obj(UniValue::VOBJ);
+    CPubKey pubkey;
+    obj.push_back(Pair("isscript", false));
+    obj.push_back(Pair("iswitness", true));
+    obj.push_back(Pair("witness_version", 0));
+    obj.push_back(Pair("witness_program", HexStr(id.begin(), id.end())));
+    return obj;
+}
+
+UniValue DescribeAddressVisitor::operator()(const WitnessV0ScriptHash& id) const
+{
+    UniValue obj(UniValue::VOBJ);
+    CScript subscript;
+    obj.push_back(Pair("isscript", true));
+    obj.push_back(Pair("iswitness", true));
+    obj.push_back(Pair("witness_version", 0));
+    obj.push_back(Pair("witness_program", HexStr(id.begin(), id.end())));
+    return obj;
+}
+
+UniValue DescribeAddressVisitor::operator()(const WitnessUnknown& id) const
+{
+    UniValue obj(UniValue::VOBJ);
+    CScript subscript;
```

Comment:
In https://github.com/bitcoin/bitcoin/pull/10583/commits/4dadd7c7ac0934447b8ca33942535cc8a22e78f5

Unused?